src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  font-family: "Roboto", Arial, sans-serif;
+  font-style: normal;
+  transition: transform, $card-transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 40px 0 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $card-text-color-accent;
+    transition: color $card-transition-duration;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__product-code {
+    margin: 4px 0 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $card-text-color-secondary;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 0;
+  }
+
+  &__reviews {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 0 0;
+
+    &-title {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $card-text-color-secondary;
+    }
+
+    &-value {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $card-text-color-accent;
+    }
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+    margin: 16px 0 0;
+    border: 1px solid $color-blue-accent;
+    border-radius: 5px;
+    height: 40px;
+    background-color: $color-blue-accent;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color-white;
+
+    transition-property: color,
+      border-color,
+      background-color;
+    transition-duration: $card-transition-duration;
```

Comment:
```suggestion
      transition-duration: $card-transition-duration;
```

Reply:
greeting from Linter:
"src/styles/blocks/card.scss
 99:7  ✖  Expected indentation of 4 spaces   indentation"